test/unit/helpers.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable no-console, import/prefer-default-export */
+
+export function display(obj) {
```

Comment:
I kept on writing this while debugging my tests.  I kept it here as a helper for that purpose.